build.gradle

```diff
@@ -19,6 +19,14 @@ allprojects {
     }
 }
 
+ext {
+    compileSdkVersion = 25
+    buildToolsVersion = '25.0.2'
+    minSdkVersion = 16
+    targetSdkVersion = compileSdkVersion
```

Comment:
`targetSdkVersion` should be separately provided a value. In this case, it happens to be equal to `compileSdkVersion` but that won't be the case always and both shouldn't be dependent on each other